src/main/java/medicaldoctor/controllers/LoginController.java

```diff
@@ -0,0 +1,83 @@
+package medicaldoctor.controllers;
+
+import java.net.URL;
+import java.util.ResourceBundle;
+import javafx.event.ActionEvent;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.PasswordField;
+import javafx.scene.control.TextField;
+import medicaldoctor.backend.data.LoginResult;
+import medicaldoctor.backend.LoginService;
+import medicaldoctor.core.AppSession;
+import medicaldoctor.entities.UserType;
+
+public class LoginController implements Initializable, ParentController{
+    ControllerManager cm;
+    
+    @FXML
+    private TextField textUsername;
+
+    @FXML
+    private PasswordField textPassword;
+
+    @FXML
+    private Button buttonSubmit;
+    
+    @FXML
+    private Label i_label;
+    
+    @FXML
+    void buttonSubmitClick(ActionEvent event) throws Exception {
+        LoginResult result = LoginService.checkLogin(
+                textUsername.getText(),
+                textPassword.getText());
+        
+        if (result == LoginResult.SUCCESS) {
+            if(AppSession.getActiveUser().hasAdditionalPassword()){
+                cm.load(LookUp.LOGIN_SECOND_SCREEN_NAME, LookUp.LOGIN_SECOND_SCREEN);
+                cm.showScreen(LookUp.LOGIN_SECOND_SCREEN_NAME);
+            }
+            else{
+                if(AppSession.getActiveUser().getType().equals(UserType.DOCTOR)){
+                    cm.load(LookUp.DOCTOR_DASHBOARD_NAME, LookUp.DOCTOR_DASHBOARD);
+                    cm.showScreen(LookUp.DOCTOR_DASHBOARD_NAME);
+                }
+                else if(AppSession.getActiveUser().getType().equals(UserType.NURSE)){
+                    cm.load(LookUp.NURSE_DASHBOARD_NAME, LookUp.NURSE_DASHBOARD);
+                    cm.showScreen(LookUp.NURSE_DASHBOARD_NAME);
+                }
+                else if(AppSession.getActiveUser().getType().equals(UserType.HEMATOLOGIC_LAB_WORKER)){
+                    cm.load(LookUp.LAB_WORKER_NAME, LookUp.LAB_WORKER);
+                    cm.showScreen(LookUp.LAB_WORKER_NAME);
+                }
+                else if(AppSession.getActiveUser().getType().equals(UserType.RADIOLOGIC_LAB_WORKER)){
+                    cm.load(LookUp.LAB_WORKER_NAME, LookUp.LAB_WORKER);
+                    cm.showScreen(LookUp.LAB_WORKER_NAME);
+                }
+                else if(AppSession.getActiveUser().getType().equals(UserType.STAFF)){  
+                    cm.load(LookUp.STAFF_DASHBOARD_NAME, LookUp.STAFF_DASHBOARD);
+                    cm.showScreen(LookUp.STAFF_DASHBOARD_NAME);
+                }
+            }
+        }
+        if (result == LoginResult.WRONG_USERNAME || result == LoginResult.WRONG_PASSWORD) {
+           i_label.setVisible(true);
```

Comment:
we should set the label to describe if it is a wrong username or password

Reply:
For security reasons shouldnt you just say incorrect login.

Reply:
not security reasons. you want to inform the user of what the problem is, not be vague. recall the UI slides. 
it would be useful to specify which is wrong so the employee can relay that to their IT department / network admin.